library/src/main/java/com/trendyol/showcase/ui/showcase/ShowcaseActivity.kt

```diff
@@ -22,6 +24,11 @@ class ShowcaseActivity : AppCompatActivity() {
                 }
             }
             setContentView(view)
+            if (model.isShowcaseViewVisibleIndefinitely.not()) {
+               Handler(Looper.getMainLooper()).postDelayed({
+                   finishShowcase(ActionType.EXIT)
```

Comment:
What if user closes the showcase before this postDelayed block called?

Reply:
There is no visible effect but the callback should be removed. Done [here](https://github.com/Trendyol/showcase/pull/26/commits/2fb95bc9041ea7aa5c774696cc436bc959a4424e)